packages/genesys-spark-components/src/components/stable/gux-popover/gux-popover.tsx

```diff
@@ -213,7 +221,17 @@ export class GuxPopover {
   }
 
   componentDidLoad(): void {
+    if (this.for) {
```

Comment:
Should we make the `for` prop required since the popover requires target button?